__tests__/account.test.ts

```diff
@@ -15,26 +21,18 @@ describe('deploy and test Wallet', () => {
   beforeAll(async () => {
     expect(account).toBeInstanceOf(Account);
 
-    const erc20Response = await provider.deployContract({
-      contract: compiledErc20,
-    });
+    const erc20DeployPayload = getERC20DeployPayload(account.address);
+
+    const erc20Response = await provider.deployContract(erc20DeployPayload);
 
     erc20Address = erc20Response.contract_address;
     erc20 = new Contract(compiledErc20.abi, erc20Address, provider);
 
     await provider.waitForTransaction(erc20Response.transaction_hash);
 
-    const mintResponse = await account.execute({
```

Comment:
sorry if I missed something, but how do we mint now the 1000 tokens? I only see this later in the test, but unrelated to this part?

Reply:
The new [ERC20 contract](https://github.com/argentlabs/argent-contracts-starknet/blob/develop/contracts/lib/ERC20.cairo) used here calls `_mint` in the constructor and mints 1000 tokens to the `recipient`. Here, `recipient` being the `account`.

Reply:
ok, maybe it would be good to add a comment about this, to be explicit? Maybe even add a link to this ERC20 contract somewhere, but not sure where, maybe in the docs?

Reply:
Yes